postback.json

```diff
@@ -0,0 +1,36 @@
+{
+    "user_id":"AB1234",
+    "unfilled_quantity":0,
+    "app_id":1234,
+    "checksum":"cbeddXXXXX",
+    "placed_by":"AB1234",
+    "order_id":"220303000308932",
+    "exchange_order_id":"1000000001482421",
+    "parent_order_id":null,
+    "status":"COMPLETE",
+    "status_message":null,
+    "status_message_raw":null,
+    "order_timestamp":"2022-03-03 09:24:25",
+    "exchange_update_timestamp":"2022-03-03 09:24:25",
+    "exchange_timestamp":"2022-03-03 09:24:25",
+    "variety":"regular",
+    "exchange":"NSE",
+    "tradingsymbol":"SBIN",
+    "instrument_token":779521,
+    "order_type":"MARKET",
+    "transaction_type":"BUY",
+    "validity":"DAY",
+    "product":"CNC",
+    "quantity":1,
+    "disclosed_quantity":0,
+    "price":0,
+    "trigger_price":0,
+    "average_price":470,
+    "filled_quantity":1,
+    "pending_quantity":0,
+    "cancelled_quantity":0,
+    "market_protection":0,
+    "meta":{},
+    "tag":null,
+    "guid":"XXXXXX"
+    }
```

Comment:
Let's format this json. Maybe pass it through `jq` once?

Reply:
I'm already using `jq pretty-print` for all. `jq . postback.json`
Somehow, while pasting from the command line, 1 extra space seems to be added at the end here. Have edited it out. Thanks